CONTRIBUTING.md

```diff
@@ -255,7 +256,17 @@ PowSyBl TSC voting members are:
 - Anne Tilloy (https://github.com/annetill)
 - Luis Zamarreno (https://github.com/zamarrenolm)
 
-Some committers are specialized in some field: please refer to [the maintainers table](MAINTAINERS.md) before submitting a pull request.
+#### Voting
+While the Powsybl organization aims to operate as a consensus based community, a TSC vote can be required to move the Organization forward. The voting members of the TSC will vote on a one vote per voting member basis. The simple majority is needed to approve proposals. A good way to vote is to create a poll [here](https://lists.lfenergy.org/g/powsybl-tsc/addpoll).
+
+#### Responsabilities
+The Powsybl organization contains several repositories. We have at least a committer in charge by repository. By "in charge", we mean:
```

Comment:
The Powsybl project comprehends several repositories. There is at least one committer in charge of each repository. By "in charge", we mean: